### DISCUSSION TITLE
Add more training to existing custom trained model

### DISCUSSION THREAD

**Discussion by osvpl** [ultralytics/ultralytics#7826] [Q&A]
I have trained my custom model using yolov8n model for epochs 100 and with approx. 300 images.
I have got best.pt and last.pt

I have few hundred more images with me now. And I want to add additional training to my above model.

How can I achieve this?

**Comment by pderrenger** [comment]
@osvpl to continue training your custom model with additional images, you can follow these steps:

1. **Prepare Your Dataset**: Combine your new images with the original dataset, ensuring that the annotations are in the correct format and the dataset is properly organized.

2. **Start Training**: Execute the training command with the updated dataset and the model checkpoint you wish to start from.

Here's an example in Python:

```python
from ultralytics import YOLO

# Load your custom trained model
model = YOLO('path/to/best.pt')  # or 'path/to/last.pt'

# Resume training with the updated dataset
results = model.train(data='updated_dataset.yaml', epochs=150)
```

And the equivalent CLI command:

```bash
yolo train data=updated_dataset.yaml model=path/to/best.pt epochs=150
```

Remember to update the `epochs` to the new total you want (e.g., if you trained for 100 epochs and want to train for another 150, set `epochs=150`). Also, ensure that your `updated_dataset.yaml` points to the directory containing both the original and the new images and annotations.

Happy training! 🚀
